pysus/online_data/__init__.py

```diff
@@ -118,3 +120,40 @@ def get_CID10_table(cache=True):
     if cache:
         df.to_parquet(cachefile)
     return df
+
+
+DB_PATHS = {'SINAN': ["/dissemin/publicos/SINAN/DADOS/FINAIS", "/dissemin/publicos/SINAN/DADOS/PRELIM"],
+            'SIM': ["/dissemin/publicos/SIM/CID10/DORES", "/dissemin/publicos/SIM/CID9/DORES"],
+            'SINASC': ["/dissemin/publicos/SINASC/NOV/DNRES", "/dissemin/publicos/SINASC/ANT/DNRES"],
+            'SIH': ["/dissemin/publicos/SIHSUS/199201_200712/Dados", "/dissemin/publicos/SIHSUS/200801_/Dados"],
+            'SIA': ["/dissemin/publicos/SIASUS/199407_200712/Dados", "/dissemin/publicos/SIASUS/200801_/Dados"],
+            'PNI': ["/dissemin/publicos/PNI/DADOS"],
+            'CNES': [f"dissemin/publicos/CNES/200508_/Dados/"],
+            'CIHA': ["/dissemin/publicos/CIHA/201101_/Dados"]
+            }
+
+
+def last_update(database: str = 'SINAN') -> pd.DataFrame:
+    """
+    Return the date of last update from the database specified.
+    :param database: Database to check
+    """
+    if database not in DB_PATHS:
+        print(f"Database {database} not supported try one of these\n{list(DB_PATHS.keys())}")
+        return pd.DataFrame()
```

Comment:
Here, it doesn't seem necessary to return *pd.Dataframe()*.
Maybe in print message, use **logging.info** instead of *print()*.

Reply:
Agree that it is not necessary to return `pd.DataFrame()`, I did it for consistency of the return type.

To use logging.info()in these cases would require setting up a logger that would write to stdout, maybe we can do this on another PR